ruby_review_week_four.rb

```diff
@@ -1,37 +1,95 @@
-# You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
+# # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
+
+# # 1. What is a variable?
+# A variable is a user-created term assigned to the value, where the value assigned may change thruoghout the program.
```

Comment:
great answer! you could have also used the word `storage` or mentioned the idea that a variable can be re-used in a program